src/Admin/AdminInterface.php

```diff
@@ -492,10 +492,21 @@ public function isChild();
      *
      * @param string $name
      *
-     * @return null|string
+     * @throws \RuntimeException
+     *
+     * @return string
      */
     public function getTemplate($name);
 
+    /**
+     * Return true if the template is set.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function hasTemplate($name);
```

Comment:
Please move both methods (and more if you find related methods) to another interface, and have `AdminInterface` extend that interface.

Reply:
It would be great to move these methods to a new `AbstractAdmin` component/collaborator

Reply:
I created a TemplateRegistry for this. Could you please review it? I'm not sure if I'm on the correct path for this. Thanks.